<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>389</width>
          <height>461</height>
         </rect>
        </property>
        <attribute name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/3.ico</normaloff>:/images/images/3.ico</iconset>
        </attribute>
        <attribute name="label">
         <string>图表设置</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>355</width>
              <height>526</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>标题</string>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLineEdit" name="editTitle"/>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="btnSetTitle">
                  <property name="text">
                   <string>设置</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QPushButton" name="btnSetTitleFont">
                  <property name="text">
                   <string>设置标题字体</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>图例</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0" colspan="2">
                 <widget class="QRadioButton" name="radioButton">
                  <property name="text">
                   <string>上方</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QRadioButton" name="radioButton_2">
                  <property name="text">
                   <string>下方</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QRadioButton" name="radioButton_3">
                  <property name="text">
                   <string>左侧</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QRadioButton" name="radioButton_4">
                  <property name="text">
                   <string>右侧</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QCheckBox" name="chkLegendVisible">
                  <property name="text">
                   <string>图例可见</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1" colspan="2">
                 <widget class="QCheckBox" name="chkBoxLegendBackground">
                  <property name="text">
                   <string>背景可见</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0" colspan="2">
                 <widget class="QPushButton" name="btnLegendFont">
                  <property name="text">
                   <string>字体</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2">
                 <widget class="QPushButton" name="btnLegendlabelColor">
                  <property name="text">
                   <string>标签颜色</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>边距</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>左边距</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="spinMarginLeft">
                  <property name="value">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>右边距</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="spinMarginRight">
                  <property name="value">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>上边距</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSpinBox" name="spinMarginTop">
                  <property name="value">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>下边距</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QSpinBox" name="spinMarginBottom">
                  <property name="value">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0" colspan="2">
                 <widget class="QPushButton" name="btnSetMargin">
                  <property name="text">
                   <string>设置边距</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>动画效果</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>动画效果</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="cBoxAnimation"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>主题</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="cBoxTheme"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>389</width>
          <height>461</height>
         </rect>
        </property>
        <attribute name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/dis3.ico</normaloff>:/images/images/dis3.ico</iconset>
        </attribute>
        <attribute name="label">
         <string>曲线设置</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>369</width>
              <height>441</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>选择操作序列</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QRadioButton" name="radioSeries0">
                  <property name="text">
                   <string>Sin曲线</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioSeries1">
                  <property name="text">
                   <string>Cos曲线</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_6">
               <property name="title">
                <string>曲线外观</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="2" column="1">
                 <widget class="QPushButton" name="btnSeriesPen">
                  <property name="text">
                   <string>曲线Pen</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QCheckBox" name="chkSeriesVisible">
                  <property name="text">
                   <string>序列可见</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>透明度</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="btnSeriesName">
                  <property name="text">
                   <string>序列名称</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QPushButton" name="btnSeriesColor">
                  <property name="text">
                   <string>曲线颜色</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QCheckBox" name="chkPointVisible">
                  <property name="text">
                   <string>数据点可见</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="editSeriesName">
                  <property name="text">
                   <string>Sin曲线</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QSlider" name="sliderSeriesOpacity">
                  <property name="maximum">
                   <number>10</number>
                  </property>
                  <property name="singleStep">
                   <number>1</number>
                  </property>
                  <property name="pageStep">
                   <number>2</number>
                  </property>
                  <property name="value">
                   <number>10</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_7">
               <property name="title">
                <string>数据点标签</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QCheckBox" name="chkPointLabelVisible">
                  <property name="text">
                   <string>数据点标签可见</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnSeriesLabColor">
                  <property name="text">
                   <string>标签颜色</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnSeriesLabFont">
                  <property name="text">
                   <string>标签字体</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_8">
                  <property name="title">
                   <string>标签格式</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QRadioButton" name="radioSeriesLabFormat0">
                     <property name="text">
                      <string>@yPoint</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radioSeriesLabFormat1">
                     <property name="text">
                      <string>(@xPoint, @yPoint)</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QChartView" name="chartView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actDraw"/>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomOut"/>
   <addaction name="actZoomReset"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actDraw">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/828.bmp</normaloff>:/images/images/828.bmp</iconset>
   </property>
   <property name="text">
    <string>刷新绘图</string>
   </property>
   <property name="toolTip">
    <string>刷新绘图</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/418.bmp</normaloff>:/images/images/418.bmp</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/416.bmp</normaloff>:/images/images/416.bmp</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="actZoomReset">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/414.bmp</normaloff>:/images/images/414.bmp</iconset>
   </property>
   <property name="text">
    <string>原始大小</string>
   </property>
   <property name="toolTip">
    <string>还原为原始大小</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
